PROTOCOL.md

```diff
@@ -424,6 +429,64 @@ enough to get you started. You can watch the data sent and received from
 the server on a regular connection, or look at the client source code
 for a full list of message types.
 
+**Action requests**
+
+These are how the client sends the player's decisions to the server. All
+requests except `/undo` can be sent with `|RQID` at the end. `RQID` is
+`REQUEST.rqid` from `|request|`. Each `RQID` is a unique number used to
+identify which action the request was intended for and is used to protect
+against race conditions with the cancel button.
+
+If an invalid request is sent, the game will replace the missing or
+erroneous request with a valid choice, which is usually the first usable
+move. Similarly, if a switch was expected but a move was sent instead,
+the game will switch in a usable Pokemon.
```

Comment:
I changed this to "a usable Pokemon" assuming that the 'not-guaranteed' problem applied here as well. If it is indeed always the first usable Pokemon let me know and I'll change it back.